### ISSUE TITLE
Spurious "&" in APA version 7 preview

### ISSUE THREAD

**Issue by crystalfp** [JabRef/jabref#8122]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

Windows

### Details on version and operating system

JabRef 5.4--2021-10-04--bac8304 Windows 10 10.0 amd64  Java 16.0.2  JavaFX 17+18

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Create a library containing the following entry:
```
@InCollection{Perlis1962,
  author    = {Perlis, Alan},
  booktitle = {{Computers and the World of the Future}},
  date      = {1962},
  title     = {The computer in the university},
  editor    = {M. Greenberger},
  location  = {Cambridge, MA},
  pages     = {180--219},
  publisher = {MIT Press},
}
```
2. Use the "American Psychological Association 7th edition" preview format
3. A spurious "&" appears in the preview:
```
Perlis, A. (1962). The computer in the university. In M. Greenberger (Ed.), & , Computers and the World of the Future (pp. 180–219). MIT Press.
```
4. This does not appears in IEEE and standard preview formats.

### Appendix

...
<details>
<summary>Log File</summary>
```
Paste an excerpt of your log file here
```
</details>

**Comment by Siedlerchr** [comment]
Hi, the CSL styles are coming from the citation styles repo and are rendered using an external library in JabRef, and from what I see others have the same problem https://github.com/michel-kraemer/citeproc-java/issues/108 
I guess it would make sense to describe the phenomen there as well
(JabRef is using the latest version)

**Comment by crystalfp** [comment]
Solved in JabRef 5.4--2021-12-14--851fa4b. Thanks!
